cypress/e2e/1-getting-started/register-wallet.cy.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference types="jest" />
+
+context('Login', () => {
+  beforeEach(() => {
+    cy.visit('/api/auth/login')
+  })
+
+// TODO: Arrumar o teste de registro de carteira
+describe('wallet Registration', () => {
+  it('should be able to register a new wallet', () => {
+    cy.contains('button', 'Continue with Microsoft Account').click()
+    cy.origin('https://login.live.com/', () => {
+      cy.get('input[type="email"]').type(Cypress.env('USERNAME'), {
+        log: false,
+      })
+      cy.get('input[type="submit"]').click()
+      cy.get('input[type="password"]').type(Cypress.env('PASSWORD'), {
+        log: false,
+      })
+      cy.get('input[type="submit"]').click()
+      cy.get('#idBtn_Back').click()
+    })
+  });
+})
+})
```

Comment:
Tem sentindo nenhum esse teste!!!

Você só copiou e colocou o teste de login e colocou na criação de uma wallet.

Reply:
Sim porque essa parte faz parte do teste, ele iria se logar ir para home e depois ir para wallet , só como disse ta como TODO pois precisa arrumar a parte após o login que não to conseguindo acessar a página